core/loadpoint.go

```diff
@@ -123,6 +123,12 @@ type LoadPoint struct {
 	chargeTimer api.ChargeTimer
 	chargeRater api.ChargeRater
 
+	// wakeUp
+	activeTimer         *ActiveTimer
+	activeDuration      time.Duration // Active current duration for wakeup calls
+	vehicleWakeUpCalled bool
```

Comment:
Brauchts das? Wenn das Auto nicht startet einfach alles triggern was wir haben. Der Aufruf dafür könnte sogar in den Timer wandern der das bei Ablauf- falls nicht vorher gestoppt- einfach tun könnte.

Reply:
Du könntest auch 2 unterschiedliche Timer starten, oder ein slice of timers haben

Reply:
ja damit ich es immer wieder aufrufe und hab es auch in den Timer verschoben.